airbyte-integrations/connectors/destination-databricks/src/test-integration/java/io/airbyte/integrations/destination/databricks/DatabricksUtilTest.java

```diff
@@ -0,0 +1,47 @@
+package io.airbyte.integrations.destination.databricks;
+
+import io.airbyte.db.Database;
+import io.airbyte.db.factory.DSLContextFactory;
+import io.airbyte.integrations.destination.databricks.utils.DatabricksConstants;
+import io.airbyte.integrations.destination.databricks.utils.DatabricksDatabaseUtil;
+import java.sql.SQLException;
+import java.util.Map;
+import org.jooq.DSLContext;
+import org.jooq.SQLDialect;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DatabricksUtilTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(DatabricksUtilTest.class);
+
+  /**
+   * Magic Databricks JDBC param. This param exists because of issue with fetching data via SELECT. It produces next error: Error occured while
```

Comment:
Hi @suhomud I am working on writing databricks source. I also faced this error. It got resolved by adding jvm args `'--add-opens=java.base/java.nio=ALL-UNNAMED'` in `build.gradle`.
Ex: 
```   
applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0', '--add-opens=java.base/java.nio=ALL-UNNAMED']
```